src/inverseRobot.js

```diff
@@ -7,7 +7,18 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const objEntries = Object.entries(robot);
```

Comment:
```suggestion
  const robotObjectEntries = Object.entries(robot);
```
give variables more descriptive names